README.md

```diff
@@ -37,8 +37,9 @@ Libraries in different languages may be in different states of development. We a
 | Objective-C             | [src/objective-c](src/objective-c)  | 1.0     |
 
 <small>
-Java source code is in the [grpc-java](http://github.com/grpc/grpc-java) repository.
-Go source code is in the [grpc-go](http://github.com/grpc/grpc-go) repository.
+Java source code is in the <a href="//github.com/grpc/grpc-java">grpc-java</a>
```

Comment:
It does not work within `<small>` tag on GitHub (check on the https://github.com/grpc/grpc rendered markdown)